src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionPatchFilterTest.java

```diff
@@ -48,11 +48,11 @@ public void testNeverSuppressedChecks() throws Exception {
 
     @Test
     public void testFileLength() throws Exception {
-        testByConfig("FileLength/newline/defaultContextConfig.xml");
```

Comment:
folder names also should stay the same

Reply:
sorry, I am not clear, current version is not same?

Reply:
I mean do not remove strategy name from the folder name. It should be consistent with other tests - `<Check>/<strategy>`. I would not rename any of these resources.
If you still want to group them under neversuppressed folder, then do this under strategy folder.
Example:
```
FileLength/newline/neversuppressed/defaultContextConfig.xml
FileLength/patchedline/neversuppressed/defaultContextConfig.xml
```

Reply:
done